Changes.rst

```diff
@@ -1,3 +1,19 @@
+Changes for v0.1.0 (2017-06-27)
+===============================
+
+-  Demo version pre-release
+
+Command line interface that interacts with the Human Cell Atlas data
+store REST API.
+
+Current functionality: Retrieve or register a data file. Retrieve or
+register a data bundle (collection of data files). Upload full directory
+to cloud, register given files, and collect them into a bundle. Download
+a bundle or file from HCA data store to local.
+
+Changes for v1.0.0 (2017-06-26)
+===============================
```

Comment:
FYI, the release script (in common.mk) uses the Git history to automatically generate a changelog for you. It then prepends the changelog to this file under the heading that corresponds to the new version (auto-generated). At the time of the release, you can edit the changelog and put this text in there, but the text that you've added here won't actually show up in the release notes on PyPI or Github.